Automattic/wp-calypso
# Domains: Show NS warning for Atomic sites and facilitate Domain Connect

#### Changes proposed in this Pull Request

This PR enables the nameserver warnings for domains mapped to Atomic sites. Also this PR emphasizes ability to connect domain via Domain Connect when it is supported.

#### Testing instructions

- Map a domain to an Atomic site and verify you see the nameserver warning/notice
- Map a domain that supports Domain Connect, and verify you see something similar to this screenshot:

<img width="726" alt="Screenshot 2020-06-02 at 9 49 30 PM" src="https://user-images.githubusercontent.com/13062352/83563375-4e9f8a80-a51b-11ea-8b64-573b1268bce6.png">
